databases/week1/hyf_lesson1.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(title) FROM task;
+SELECT*FROM task WHERE due_date IS NULL;
+SELECT task.id, task.title, status.name  FROM task, status WHERE status.name LIKE "%Done%";
+SELECT task.id, task.title, status.name FROM task , status WHERE status.name NOT LIKE "%Done%";
```

Comment:
Line 4. Task: _Find all the tasks that are not marked as done_
Like in the previous case, you will have to find a way to work with 2 tables. 
This code also brings back duplicated and mixed ids 🙏